tools/java/org.hl7.fhir.tools.core/src/org/hl7/fhir/tools/implementations/go/MgoModel.java

```diff
@@ -560,6 +559,10 @@ public String getMongoField() {
           imports.add("fmt");
         }
 
+        if (name.equals("Bundle") || name.equals("Parameters")) {
+            imports.add("gopkg.in/mgo.v2/bson");
+        }
```

Comment:
This is kind of brittle -- if FHIR adds a "Resource" field to other resource types, things will break and you'll have to manually update this.  Luckily, I *think* you just need to add this import in line 535 above (I think the conditions are the same).  Please try it -- because that would be more future-proof if it works.